db/1.13/modules/ROOT/pages/database-connector-xa-transactions.adoc

```diff
@@ -0,0 +1,44 @@
+= Using XA Transactions in Database Connector
+
+Anypoint Connector for Database (Database Connector) provides the option to use Extended Architecture Transactions (XA Transactions) when you enable the *Use XA Transactions* field in the Database Connector global configuration.
+
+== Before You Begin
+
+Before getting started, ensure that you have understanding of xref:mule-runtime::xa-transactions.adoc[XA Transactions]
+
+== Use XA Transactions
+
+You can enable the *Use XA Transactions* field in the database global connector configuration for the Derby Connection, Generic Connection, Microsoft SQL Server Connection, MySQL Connection and Oracle Connection.
+For a DataSource connection, refer to the xref:database-connector-connection.adoc#data-source-connection-xa[Configure a Data Source Reference Connection When in XA Transaction Mode] documentation. 
+
+[NOTE]
+When you enable *Use XA Transactions* the connection pooling is created as well. 
+
+The following example shows how to configure *Use XA Transactions* for *MySQL Connection* in Anypoint Studio:
```

Comment:
```suggestion
The following example shows how to configure *Use XA Transactions* for the *MySQL Connection* in Anypoint Studio:
```

Reply:
You didn't bold the connection names earlier - either way, but make them consistent